tests/lean/eval_except.lean

```diff
@@ -0,0 +1,5 @@
+prelude
+import Init.System.IO
+
+#eval (throw $ IO.userError "this is my error" : IO Unit)
+#eval (throw $ IO.Error.noFileOrDirectory "file.ext" "and details" : IO Unit)
```

Comment:
perhaps add a comment here that "and details" is not expected to print

Reply:
Maybe I should just delete details from those that don't print